packages/python/plotly/plotly/express/_doc.py

```diff
@@ -111,6 +111,10 @@
         colref_list,
         "Values from these columns appear as extra data in the hover tooltip.",
     ],
+    custom_data=[
+        colref_list,
+        "Values from these columns are extra data, to be used in widgets or Dash callbacks for example.",
```

Comment:
Maybe a note that this data is not user-visible, but is included in events emitted by the figure?